contracts/SignHash.sol

```diff
@@ -14,6 +15,7 @@ contract SignHash {
     //--- Events
 
     event HashSigned(bytes32 indexed hash, address indexed signer);
+    event SignerProved(address indexed signer, string method, string value);
```

Comment:
I wanted to have succinct and consistent names. Not necessarily over descriptive.
The logic is:
- transactions: `sign`, `prove`
- accessors: `getSigners`, `getProof`

And here comes events:
- `HashSigned` - what - `hash`, operation: `sign()`
- `SignerProved` - what - `signer`, operation: `prove()`

I am open to changes but I think that `SignerProveAdded` is incorrect (prove/proof) and too long. `ProofAdded` seems ok, in case of transaction `revoke` we would have `ProofRemoved`. But to be consistent I would also consider changing `HashSigned` to `SignerAdded`.

Reply:
After thinking about it I don't like `SignerAdded` - it is not clear. `HashSigned` should stay. 
What would you say for following list of events and corresponding transactions:
- `HashSigned` - `sign()`
- `ProofAdded` - `prove()`
- `ProofRemoved` - `revoke()`

Or maybe we should change `prove()` and `revoke()` functions to `addProof()` and `removeProof()`?

Reply:
I agree with `addProof` and `removeProof` those look less cryptic than `prove` and `revoke`, also they would match `ProofAdded/Removed` events obviously. 
As for `HashSigned` I have mixed feelings, the name is good but not also entirely consistent with the others. I don't have any better ideas, so let's keep to your suggestion and have `HashSigned`, we will adjust it if needed. My last comment might have been lost by I'm also +1 to `ProofAdded` and `ProofRemoved`.